components/MarketClosureIcon/MarketClosureIcon.tsx

```diff
@@ -21,44 +23,20 @@ export const MarketClosureIcon: FC<MarketClosureIconProps> = ({
 	const sharedProps = {
 		width: size === 'sm' ? 16 : 32,
 		height: size === 'sm' ? 16 : 32,
-		className: 'market-closure-icon',
+		// className: 'market-closure-icon',
```

Comment:
I think we can safely either delete this line or uncomment it and remove the className prop on Line 29